learning-pathways/io5.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: learning-pathway
+tags: [beginner]
+type: use
+
+editorial_board:
+- bebatut
+- fpsom
+funding:
+- erasmusplus
+
+title: Gallantries Grant - Intellectual Output 5 - Train-the-Trainer and mentoring programme
+
+cover-image: ./shared/images/Gallantries_logo.png
+cover-image-alt: "Gallantries logo with the carpentries wrench in galaxy 2 stripes 1 strip colour scheme."
+
+priority: 5
+draft: true
+
+
+
+pathway:
+
+  - section: "Hybrid Lesson Development Guide"
+    description: |
+      Covers an introduction to pedagogy for instructors and how to adapt training materials to hybrid methodology and key points and indicators that instructors should use when evaluating or adapting training materials to hybrid training delivery [SC5.1]
+    tutorials:
+      - name:
+        topic: 
```

Comment:
@shiltemann will take care of the updates for this one (maybe as FAQs in the contributing tutorial)